app/models/ncr/work_order.rb

```diff
@@ -28,21 +29,56 @@ class WorkOrder < ActiveRecord::Base
       greater_than_or_equal_to: 0,
       message: "must be greater than or equal to $0"
     }
+    validates :cl_number, format: {
+      with: /\ACL\d{7}\z/,
+      message: "must start with 'CL', followed by seven numbers"
+    }, allow_blank: true
     validates :expense_type, inclusion: {in: EXPENSE_TYPES}, presence: true
+    validates :function_code, format: {
+      with: /\APG[A-Z0-9]{3}\z/,
+      message: "must start with 'PG', followed by letters or numbers"
```

Comment:
For consistency, should this say "3 letters or numbers"? Or "characters" to match `soc_code` below? "Characters" isn't exactly accurate, though.